src/layouts/components/CoursesContainer.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import styled from 'styled-components'
+import { withTheme } from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { media } from '../../theme/globalStyle'
+import FilterAndSearch from './FilterAndSearch'
+import JobsList from './JobsList'
+import Pagination from './Pagination'
+import { fakeLearnAPI, fakeLearnAPISearch } from '../utils/api'
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: auto;
+  grid-column-gap: 1rem;
+  grid-template-areas:
+    '. . fs fs fs fs fs fs fs fs . .'
+    '. . lst lst lst lst lst lst lst lst . .';
+  margin-top: 2.5rem;
+  ${media.giant`
+    grid-column-gap: 0.5rem;
+    grid-template-areas:
+      '. fs fs fs fs fs fs fs fs fs fs .'
+      '. lst lst lst lst lst lst lst lst lst lst .';
+  `} ${media.tablet`
+    grid-template-areas:
+      'fs fs fs fs fs fs fs fs fs fs fs fs'
+      '. lst lst lst lst lst lst lst lst lst lst .';
+  `} ${media.phone`
+    grid-template-areas:
+      'fs fs fs fs fs fs fs fs fs fs fs fs'
+      '. lst lst lst lst lst lst lst lst lst lst .';
+  `};
+`
+
+const List = styled.div`
+  grid-area: lst;
+  display: flex;
```

Comment:
Using flexbox and CSS grid now?

Reply:
Yes ☺
I find it handy to use both, especially for responsive design. CSS Grid for top-level structures and Flex for the lower-lever.